evalml/tests/data_checks_tests/test_data_checks.py

```diff
@@ -216,7 +218,13 @@ def test_default_data_checks_classification(input_type):
 
     y = pd.Series([0, 1, np.nan, 1, 0])
     y_multiclass = pd.Series([0, 1, np.nan, 2, 0])
-    X.ww.init(logical_types={"natural_language_nan": "NaturalLanguage"})
+    X.ww.init(
+        logical_types={
```

Comment:
Nice, I think this covering the default data checks gives us coverage for a lot of the data checks.